Multisig/UI/ClaimToken/Select Guardian/GuardianListViewController.swift

```diff
@@ -148,12 +141,12 @@ extension GuardianListViewController: UISearchResultsUpdating, UISearchBarDelega
         }
         resultsController.selectedDelegate = selectedDelegate
         resultsController.tableView.reloadData()
+        Tracker.trackEvent(.userClaimChdelSearch)
```

Comment:
here you can switch between this event and `screenClaimChdelNf`:
```suggestion
        Tracker.trackEvent(resultsController.filteredGuardians.isEmpty ? .screenClaimChdelNf : .userClaimChdelSearch)
```

Reply:
We do want to track `userClaimChdelSearch` even if there is no result, nor?